streams/us.m3u

```diff
@@ -249,7 +249,7 @@ https://live.cnbcindonesia.com/livecnbc/smil:cnbctv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="CNCTV.us",County News Center (CNC TV) (San Diego CA) (720p)
 https://cdn3.wowza.com/5/V2Y2VmhqMEFDTUkx/sdcounty/G0283_012/playlist.m3u8
 #EXTINF:-1 tvg-id="CNN.us",CNN
-https://16live00.akamaized.net/CNN/index.m3u8
+https://tve-live-lln.warnermediacdn.com/hls/live/586495/cnngo/cnn_slate/VIDEO_0_3564000.m3u8
```

Comment:
Offline.
<img width="692" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/ac5aaedb-7e8d-492b-831f-f25909a000e9">

Reply:
```suggestion
https://tve-live-ctl.warnermediacdn.com/hls/live/586495/cnngo/cnn_slate/VIDEO_0_3564000.m3u8
```

Reply:
Did you try it via USA IP? It is geoblocked